app/controllers/sessions_controller.rb

```diff
@@ -7,9 +7,18 @@ class SessionsController < Devise::SessionsController
   # before_action :configure_sign_in_params, only: [:create]
 
   # GET /resource/sign_in
-  # def new
-  #   super
-  # end
+  def new
```

Comment:
**nitpick (blocking):** could you refactor this to call `super` and pass your render logic in as block, this way we know our controller won't have to be updated if this gets updated in Devise.

e.g.
```ruby
def new
  super do
     # custom view logic
     return
  end
end
```

The `yield resource if block_given?` is in the devise controller so that you can do the above.

Reply:
Please see 6fddb0f89573a2c6e2811c66a37cd4ea05ce707f.